NEWS.md

```diff
@@ -0,0 +1,42 @@
+# Changes by Version
```

Comment:
We should also add here that we're following Semantic Versioning, and maybe reference/give credit to "keep a changelog"?

Reply:
That should go in a CONTRIBUTING.md which we don't have for some reason. This is what I have in Electron Packager's contributing docs:

> If you are contributing a nontrivial change, please add an entry to NEWS.md. The format is similar to the one described at [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).

Reply:
ok, I like the idea of having a CONTRIBUTING.md

Reply:
Added #120 for this.